bioio_ome_zarr/writers/__init__.py

```diff
@@ -1,23 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from .channel import Channel
 from .ome_zarr_writer_v2 import OMEZarrWriter as OmeZarrWriterV2
 from .ome_zarr_writer_v2 import (
     chunk_size_from_memory_target,
     compute_level_chunk_sizes_zslice,
     compute_level_shapes,
     resize,
 )
-from .ome_zarr_writer_v3 import OMEZarrWriter as OmeZarrWriterV3
-from .ome_zarr_writer_v3 import default_axes, downsample_data
+from .ome_zarr_writer_v3 import OMEZarrWriterV3 as OmeZarrWriterV3
+from .ome_zarr_writer_v3 import spatial_downsample
```

Comment:
do we still need default_axes?  I see it on line23

Reply:
No this opts to use the Axes object instead.

Reply:
resolved in [ffaa86e](https://github.com/bioio-devs/bioio-ome-zarr/pull/42/commits/ffaa86ea4868aed27a7b33259346314a41799a89)****